src/java/org/apache/cassandra/io/FSErrorHandler.java

```diff
@@ -27,4 +27,5 @@
 {
     void handleCorruptSSTable(CorruptSSTableException e);
     void handleFSError(FSError e);
+    default void handleStartupFSError(Throwable t) {}
```

Comment:
Do we need a `default` implementation here?

Reply:
its in case someone overrides, we don't break compatibility.